tools/Dockerfile

```diff
@@ -1,70 +1,47 @@
-# syntax=docker/dockerfile:1.1.3-experimental
+# syntax=docker/dockerfile:1.4.3
 # This Dockfile contains separate targets for each testsys agent
-# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
-# Shared build stage used to build Go binaries.
 ARG BUILDER_IMAGE
-FROM ${BUILDER_IMAGE} as build-go
-USER builder
+FROM "${BUILDER_IMAGE}" as base
+
+USER root
+COPY ./hashes /hashes
+COPY ./clarify.toml /clarify.toml
+
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+# Shared build stage used to build Go binaries.
+FROM base as build-go
 
 ARG GOARCH
 ARG GOOS=linux
 ARG GOROOT="/usr/libexec/go"
 ARG GOPROXY
 
+USER builder
```

Comment:
Resets back to root on line 26, and no actions happen between here and then, so this looks unnecessary.

Reply:
That should be adding the the proper **GOROOT** and **GOPROXY** to `builder`'s environment, which will get shared out to everything calling `build-go`, but it may be that the **ENV** would get shared regardless.